src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController.php

```diff
@@ -653,6 +692,54 @@ public function resendEmailAction(int $orderId, int $orderStatusId, int $orderHi
         ]);
     }
 
+    public function deleteProductAction(int $orderId, int $orderDetailId): JsonResponse
+    {
+        $this->getCommandBus()->handle(
```

Comment:
Can we please handle the usecase where the command fails 🙏 and throws an exception ? with a try/catch

Since here you are returning a JSON, I think we should return a HTTP 400 JSON response which contains an error message. Does this looks relevant ? 🤔

Reply:
:ballot_box_with_check: Thanks & Good idea